### DISCUSSION TITLE
Can I use Pysph to simulate the underwater explosion bubble pulsation problem?

### DISCUSSION THREAD

**Discussion by cqkinteresting** [pypr/pysph#374] [Q&A]
Involves gas-liquid two-phase flow

**Comment by nauaneed** [comment]
It would have been helpful if you had given more details about this problem, or pointed to some literature that specifically describes what you intend to simulate. 

Multiphase flows may require some work. Not all schemes in pysph are written with multiphase flow in mind. For example, some would use the same $\gamma$(specific heat ratio) in the equation of state for all fluids. Issues like these could be worked around easily. You may also need to implement some equations. Other than that, I think pysph should be capable of multiphase flows without any **fundamental** changes.
